code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -37,7 +37,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
 91-100: Dangerously toxic - swift death
 */
 
-/datum/reagent/consumable/ethanol/New()
+/datum/reagent/consumable/ethanol/New(list/data)
+	if(data)
```

Comment:
LAZYLEN?

Reply:
I have no clue what's the difference, but sure.

Reply:
It also checks if your list isnt empty